examples/getFollowing.js

```diff
@@ -1,18 +1,7 @@
-
-
 const GitHubApi = require('github')
 
-
 const github = new GitHubApi({
   debug: true
 })
 
-github.authenticate({
-  type: 'oauth',
-  token: 'add-your-real-token-here'
-})
```

Comment:
oh sorry I only now realised you removed these. We have to keep these in for all request that edit something, the invalid token here is important, the examples will fail and we expect them. But if we leave that out and I configure a token for my own account that has actual access to the `octokit/node-github` repository, I can end up changing stuff that I don't want to change :)

Reply:
Lol I accidentally invited @defunkt because of that, sorry Chris :)